src/avatar/avatar.component.ts

```diff
@@ -69,6 +75,10 @@ export class ThyAvatarComponent implements OnInit {
 
     @Input() thyDisabled: boolean;
 
+    @Input() thyLoading?: ThyAvatarLoading;
+
+    @Input() thyFetchPriority?: ThyAvatarFetchPriority;
```

Comment:
Add test cases for attributes loading and fetchpriority

Reply:
Done